src/auto-evo/RunResults.cs

```diff
@@ -169,12 +169,18 @@ string PatchString(Patch patch)
 
             void OutputPopulationForPatch(Species species, Patch patch, long population)
             {
-                builder.Append("  ");
-
-                builder.Append(PatchString(patch));
-
-                builder.Append(" population: ");
-                builder.Append(Math.Max(population, 0));
+                if (population > 0)
+                {
+                    builder.Append("  ");
+                    builder.Append(PatchString(patch));
+                    builder.Append(" population: ");
```

Comment:
I think you removed the population print?

Reply:
In my defense, my father did drop me on my head as a child.